examples/Python/ImportPythonModel.ipynb

```diff
@@ -29,9 +29,7 @@
     "3. Write JSON files that are associated with the trained model and write the model score code .py file. Also, write JSON files for one of the following data options:\n",
     "   (a) Generate Fit Statistics from user-defined input.\n",
     "   (b) Calculate Fit Statistics, ROC curve and Lift information from data.\n",
-    "4. Write a score code Python file for model scoring in SAS Model Manager or SAS Model Manager.\n",
-    "5. Zip the pickle, JSON, and score code files into an archive file.\n",
-    "6. Import the ZIP archive file to SAS Model Manager or SAS Model Manager via an API call."
+    "4. Import model into SAS Model Manager using an import_model call. This will generate the necessary score code as well as ZIP the model and send it to SAS Model Manager."
```

Comment:
Text needs a slight modification.
4. Import model into SAS Model Manager using an import_model call. This call generates the necessary score code and creates a ZIP archive file for the model and then sends it to SAS Model Manager."